infrastructure/tooling/src/build/tasks/common.js

```diff
@@ -33,7 +33,7 @@ module.exports = ({tasks, cmdOptions, credentials, baseDir, logsDir}) => {
       });
 
       return {
-        'docker-flow-version': {
+        'docker-flow-version': JSON.stringify({
```

Comment:
I supposed this will fix `version.json containing [Object object]`?

Reply:
I hope so :)